### PR TITLE
code update for sellerConfig,[ATPRule +SafetyStock=inventoryAvailabil…

### PR DESCRIPTION
**Pull request by sheeladubey**
code update for sellerConfig,[ATPRule +SafetyStock=inventoryAvailability],inventoryOrganization,Returnitems

### CODE REVIEW

**Comment by sheeladubey** [comment]
code update for sellerConfig,[ATPRule +SafetyStock=inventoryAvailability],inventoryOrganization,Returnitems

File: src/main/java/com/gsicommerce/romui/selenium/pages/ordermanagement/CommonElementsPage.java [hunk 1]
```diff
@@ -178,9 +174,9 @@ public static int getTotalRows() throws Exception {
 		return nodeWebTable().GetNumOfRows();
 	}
 
-	public static void clickSelectLink(int row, int col, int divchild, int buttonchild) throws Exception {
+	public static void clickSelectLink(int row, int col) throws Exception {
```
Comment (kothapallig) [inline src/main/java/com/gsicommerce/romui/selenium/pages/ordermanagement/CommonElementsPage.java:177]: Is it like you reverted the original code here???

File: src/main/java/com/gsicommerce/romui/selenium/utilities/Common.java [hunk 2]
```diff
@@ -586,22 +578,4 @@ public static void clickonCalendarDate(WebElement el, int days) {
 			el.sendKeys(selectDate(days));
```
Comment (kothapallig) [inline src/main/java/com/gsicommerce/romui/selenium/utilities/Common.java:578]: from where el is read?

Comment (sheeladubey) [reply src/main/java/com/gsicommerce/romui/selenium/utilities/Common.java:578]: method clickonCalendarDate() already reviewed in last review  session.el is getting called in pipelinepage.java file.
New method need to review is waitForPageLoaded().

**Comment by sheeladubey** [comment]
gopi added new methed waitForPageLoaded() in common.java.this is change in common code.Also added new methods like clickOnCancelBtn and clickDivChildSpanLink and modified clickDivSpanLink in CommonElementsPage.java

File: src/main/java/com/gsicommerce/romui/selenium/pages/ordermanagement/CommonElementsPage.java (hunk 1 above)
Comment (sheeladubey) [reply src/main/java/com/gsicommerce/romui/selenium/pages/ordermanagement/CommonElementsPage.java:177]: no gopi.Created new branch with name -sellerConfig,inventoryAvailability,inventoryOrganization and pushed changes.I have made changes in CommonElementsPage.java file- method clickSelectLink .I have put else condition here to work for my code.Please review and let me know 

public void clickButton(int row, int col,int divchild,int buttonchild) {
		if(divchild==0 && buttonchild==0) {
		_xpath = String.format(".//tr[" + row + "]/td[" + col + "]/div/button", row, col);
		
		}else
		{
			_xpath = String.format(".//tr[" + row + "]/td[" + col + "]/div["+divchild+"]/button["+buttonchild+"]", row, col);
		}

**Comment by sheeladubey** [comment]
Gopi-Common.java file having -waitForPageLoaded() method  added newly.
CommonElementPage.java having clickOnCancelBtn and clickDivChildSpanLink  added newly and clickSelectLink() modified with else statement .

**Pull request merged**